README.md

```diff
@@ -477,13 +477,10 @@ In this case, we can just change the override to point to some commit of our for
 
 
 ```haskell
-let overrides =
-    { simple-json =
-          upstream.simple-json
-       // { repo = "https://github.com/my-user/purescript-simple-json.git"
-          , version = "701f3e44aafb1a6459281714858fadf2c4c2a977"
-          }
-    }
+let upstream = -- <package set URL here>
+in  upstream
+  with simple-json.repo = "https://github.com/my-user/purescript-simple-json.git"
+  with simple-json.verison = "701f3e44aafb1a6459281714858fadf2c4c2a977"
```

Comment:
Typo here -- `version`

Reply:
Dah! I'm pretty sure my fingers have muscle memory now in spelling this word wrong!

Reply:
Fixing in #662